<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorSessionForm</class>
 <widget class="QDialog" name="qtractorSessionForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>640</width>
     <height>320</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Session</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qtractor.qrc">:/images/qtractor.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="SessionTabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="SessionTab" >
      <attribute name="title" >
       <string>Session</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="SessionNameTextLabel" >
         <property name="text" >
          <string>&amp;Name:</string>
         </property>
         <property name="buddy" >
          <cstring>SessionNameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SessionNameLineEdit" >
         <property name="minimumSize" >
          <size>
           <width>320</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Session name</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="SessionDirTextLabel" >
           <property name="text" >
            <string>&amp;Directory:</string>
           </property>
           <property name="buddy" >
            <cstring>SessionDirComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="AutoSessionDirCheckBox">
           <property name="toolTip">
            <string>Whether to auto-name the session directory</string>
           </property>
           <property name="text">
            <string>&amp;Auto</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="SessionDirComboBox" >
           <property name="minimumSize" >
            <size>
             <width>320</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Session directory</string>
           </property>
           <property name="editable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="SessionDirToolButton" >
           <property name="minimumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="icon" >
            <iconset theme="fileOpen" />
           </property>
           <property name="focusPolicy" >
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="toolTip" >
            <string>Browse for session directory</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="DescriptionTextLabel" >
         <property name="text" >
          <string>&amp;Description:</string>
         </property>
         <property name="buddy" >
          <cstring>DescriptionTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="DescriptionTextEdit" >
         <property name="toolTip" >
          <string>Session description</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PropertiesTab" >
      <attribute name="title" >
       <string>Properties</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QGroupBox" name="TimeGroupBox" >
         <property name="title" >
          <string>Time</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="SampleRateTextLabel" >
            <property name="text" >
             <string>Sample &amp;Rate:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>SampleRateComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="SampleRateComboBox" >
            <property name="toolTip" >
             <string>Sample rate (Hz)</string>
            </property>
            <property name="editable" >
             <bool>true</bool>
            </property>
            <item>
             <property name="text" >
              <string>44100</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>48000</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>96000</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>192000</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="TempoTextLabel" >
            <property name="text" >
             <string>&amp;Tempo:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>TempoSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="qtractorTempoSpinBox" name="TempoSpinBox" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Tempo (BPM) / Signature</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="TicksPerBeatTextLabel" >
            <property name="text" >
             <string>T&amp;icks/Beat:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>TicksPerBeatSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="TicksPerBeatSpinBox" >
            <property name="toolTip" >
             <string>Resolution (ticks/beat; tpqn)</string>
            </property>
            <property name="minimum" >
             <number>24</number>
            </property>
            <property name="maximum" >
             <number>3840</number>
            </property>
            <property name="singleStep" >
             <number>24</number>
            </property>
            <property name="value" >
             <number>960</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ViewGroupBox" >
         <property name="title" >
          <string>View</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="SnapPerBeatTextLabel" >
            <property name="text" >
             <string>&amp;Snap/Beat:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>SnapPerBeatComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="SnapPerBeatComboBox" >
            <property name="toolTip" >
             <string>Snap/beat</string>
            </property>
            <property name="editable" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="PixelsPerBeatTextLabel" >
            <property name="text" >
             <string>&amp;Pixels/Beat:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>PixelsPerBeatSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="PixelsPerBeatSpinBox" >
            <property name="toolTip" >
             <string>Pixels/beat</string>
            </property>
            <property name="minimum" >
             <number>4</number>
            </property>
            <property name="maximum" >
             <number>240</number>
            </property>
            <property name="singleStep" >
             <number>4</number>
            </property>
            <property name="value" >
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="HorizontalZoomTextLabel" >
            <property name="text" >
             <string>&amp;Horizontal Zoom:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>HorizontalZoomSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="HorizontalZoomSpinBox" >
            <property name="toolTip" >
             <string>Horizontal Zoom (%)</string>
            </property>
            <property name="suffix" >
             <string> %</string>
            </property>
            <property name="minimum" >
             <number>10</number>
            </property>
            <property name="maximum" >
             <number>1000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="VerticalZoomTextLabel" >
            <property name="text" >
             <string>&amp;Vertical Zoom:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>VerticalZoomSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QSpinBox" name="VerticalZoomSpinBox" >
            <property name="toolTip" >
             <string>Vertical Zoom (%)</string>
            </property>
            <property name="suffix" >
             <string> %</string>
            </property>
            <property name="minimum" >
             <number>10</number>
            </property>
            <property name="maximum" >
             <number>1000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <customwidgets>
  <customwidget>
   <class>qtractorTempoSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtractorSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SessionNameLineEdit</tabstop>
  <tabstop>AutoSessionDirCheckBox</tabstop>
  <tabstop>SessionDirComboBox</tabstop>
  <tabstop>SessionDirToolButton</tabstop>
  <tabstop>DescriptionTextEdit</tabstop>
  <tabstop>SampleRateComboBox</tabstop>
  <tabstop>TempoSpinBox</tabstop>
  <tabstop>TicksPerBeatSpinBox</tabstop>
  <tabstop>SnapPerBeatComboBox</tabstop>
  <tabstop>PixelsPerBeatSpinBox</tabstop>
  <tabstop>HorizontalZoomSpinBox</tabstop>
  <tabstop>VerticalZoomSpinBox</tabstop>
  <tabstop>SessionTabWidget</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc" />
 </resources>
 <connections/>
</ui>
